DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Reports/_ActivityTable.cshtml

```diff
@@ -1,5 +1,11 @@
 ﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Centre.Reports
 @model UsageStatsTableViewModel
+@{
+    //These must match with the name in trackingSystem/report.ts
+    const string toggleRowButtonClass = "js-toggle-row-button";
+    const string toggleActivityRowClass = "js-toggleable-activity-row";
+    const int defaultNumOfRowsVisible = 5;

```

Comment:
Two things here:
The names don't match the names of variables in reports.ts - it would be helpful if they did, otherwise future devs will have to work harder to know which ones should match. I prefer these names as the syntax is more similar to English. Though I would change toggleActivityRowClass to toggl**able**ActivityRowClass.

It currently looks like `defaultNumOfRowsVisible` should match a variable on reports.ts. I'd just move it above the comment to fix that.